packages/model/src/calendar.ts

```diff
@@ -57,6 +57,8 @@ export interface CalendarDataObject {
   resourceId?: string | null;
   expirationDate?: number | null;
   version: number;
+  group?: boolean;
```

Comment:
this doesn't look right - first of all an optional boolean makes no sense to me. Second - are we sure we don't just want to return the list of groups and workingGroups associated with it? Weren't you supposed to remove the calendars from the list of working-groups which are inactive?